### DISCUSSION TITLE
Make ANSI background color span over full line height

### DISCUSSION THREAD

**Discussion by n-peugnet** [community/community#24976] [Actions]
<p>I would like that, in the log web view, the ansi background colors filled the entire height of a line.</p>
<p>This could be done quite easily, by adding the following css line to each <code>ansibg</code> <code>span</code>:</p>
<pre><code class="lang-css">padding: 2px 0;
</code></pre>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/f/df6c1371cac9d6fbd8dabbdfd9437bb5d412cf96.gif" data-download-href="/uploads/short-url/vSu1I0891kL5CnyZuPIriTixVae.gif?dl=1" title="Peek 2021-01-16 17-37" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/23519418/181087971-b8ea1db4-c4c7-4554-8e55-b385e29dd073.gif" alt="Peek 2021-01-16 17-37" data-base62-sha1="vSu1I0891kL5CnyZuPIriTixVae" width="690" height="258" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/f/df6c1371cac9d6fbd8dabbdfd9437bb5d412cf96_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Peek 2021-01-16 17-37</span><span class="informations">1010×378 2.44 MB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by thyeggman** [comment]
<p>Hi there! A change to increase the background color height to the full line height was just pushed to the site today.  Let us know what you think!</p>

**Comment by n-peugnet** [comment]
<p>This change has introduced a small bug on my device : as the background is one pixel higher than the line, a scrollbar is being displayed.<br>
I fixed this by using a <code>3px</code> padding, but this may not be the best solution.<br>
Maybe hiding the overflow would be a better idea, but using <code>overflow: hidden</code> didn’t do the trick on my device.<br>
</p><div class="large-image-placeholder"><a href="https://public.club1.fr/nicolas/Peek%202021-02-12%2015-09.gif" target="_blank" rel="noopener nofollow ugc"><svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="url">https://public.club1.fr/nicolas/Peek%202021-02-12%2015-09.gif</span><span class="help">(image larger than 4096KB)</span></a></div>

**Comment by thyeggman** [comment]
<p>What device are you using?  What browser are you using and which version is it?</p>

**Comment by thyeggman** [comment]
<p>I was able to reproduce this on Safari, but it looks like you’re not using Safari either.  Let me know what you’re using and I’ll work on a fix in the meantime</p>

**Comment by n-peugnet** [comment]
<p>I’m using Firefox 85.0.1 (64-bit) on Linux, at 100% zoom, with a classic HD screen resolution.</p>

**Comment by thyeggman** [comment, marked as answer]
<p>Hi there, the issue should be fixed now!  Thanks for your report and additional information you provided to help :slightly_smiling_face:</p>

**Comment by n-peugnet** [comment]
<p>Hi, thank you it is (almost) perfect ! Sorry for the nitpick but I still have one very small issue: It really is not that important, but empty background colored lines seems to have disappeared (see previous images for comparison).<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/8/781c52a0fe892fd7a90070f45c609bdc37aac6c2.png" data-download-href="/uploads/short-url/h8xYCiTQ38tK1eBxIFoXtzr8Y4a.png?dl=1" title="2021-02-17-205422_944x319_scrot" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/23519418/181087976-7aa8431b-8214-4d18-996a-bf1f5ee22a04.png" alt="2021-02-17-205422_944x319_scrot" data-base62-sha1="h8xYCiTQ38tK1eBxIFoXtzr8Y4a" width="690" height="233" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/8/781c52a0fe892fd7a90070f45c609bdc37aac6c2_2_690x233.png, https://aws1.discourse-cdn.com/github/original/3X/7/8/781c52a0fe892fd7a90070f45c609bdc37aac6c2.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/8/781c52a0fe892fd7a90070f45c609bdc37aac6c2.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/8/781c52a0fe892fd7a90070f45c609bdc37aac6c2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2021-02-17-205422_944x319_scrot</span><span class="informations">944×319 22 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>You can also look at the actual run if you want to check yourself:</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/vincent-peugnet/antilope/runs/1714146687?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:500/500;"><img src="https://user-images.githubusercontent.com/23519418/181087978-daeca2bf-3550-40cd-bcd5-97ed9662ec72.png" class="thumbnail" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png"></div>

<h3><a href="https://github.com/vincent-peugnet/antilope/runs/1714146687?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">Build software better, together</a></h3>

<p><a href="https://github.com/vincent-peugnet/antilope/runs/1714146687?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">nt-peugnet/antilope/runs/1714146687</a></p>

  <p><span class="label1">GitHub is where people build software. More than 56 million people use GitHub to discover, fork, and contribute to over 100 million projects.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by thyeggman** [comment]
<p>Thanks for the report, we’ll create an issue for it.  Your case is pretty unique, coloring lines full of spaces, so it’s something we didn’t cover in our testing.</p>
